packages/components/src/components/link-button/shadow.tsx

```diff
@@ -54,11 +55,10 @@ export class KolLinkButton implements LinkButtonProps, FocusableElement {
 			<Host class="kol-link-button">
 				<KolLinkWcTag
 					ref={this.catchRef}
-					class={{
-						button: true,
-						[this._variant as string]: this._variant !== 'custom',
+					class={clsx('kol-link-button', {
```

Comment:
Die Klassen sollten wir zwecks (interner) Wiederverwenbarkeit in component.tsx verschieben.

Reply:
Meinst du in die Link component.tsx? Link-Button hat keine component.tsx.
Die customClass und die variant in die Link-Component zu verschieben wird schwierig. Link kennt keine customClass und variant. Erstmal müssten wir dann die props erweitern.